CodingEasy/urls.py

```diff
@@ -17,7 +17,15 @@
 from django.contrib import admin
 from django.urls import path, include
 
+#for favison
```

Comment:
Please correct this comment to #for favicon, to avoid any confusion for future.

Reply:
And update this comment in all the files, where its written.

Reply:
yeah sure